### PR TITLE
16609 Sum row not resized on activation

### PR DESCRIPTION
**Pull request by hschink**
[`* `#16609` Sum row not resized on activation`](https://community.openproject.org/work_packages/16609)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-16609-sum-row-not-resized-on-activation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js [hunk 1]
```diff
@@ -126,8 +126,19 @@ angular.module('openproject.workPackages.directives')
...
+        // This eventually calls the resize event handler defined in the
+        // WorkPackagesTable directive and ensures that the sum row at the
+        // table footer is properly displayed.
+        triggerResizeEvent();
       };
 
+      function triggerResizeEvent() {
+        var resizeEvent = jQuery.Event('resize');
+
+        angular.element($window).triggerHandler(resizeEvent);
```
Comment (myabc) [inline app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js]: @hschink I believe you should be able to write: `angular.element($window).trigger('resize');`

Comment (hschink) [inline app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js]: @myabc Thanks for the hint! It seems to be working even better. Beware, I amended the change.

File: app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js [hunk 2]
```diff
@@ -126,8 +126,19 @@ angular.module('openproject.workPackages.directives')
...
+        // This eventually calls the resize event handler defined in the
+        // WorkPackagesTable directive and ensures that the sum row at the
+        // table footer is properly displayed.
+        triggerResizeEvent();
       };
 
+      function triggerResizeEvent() {
+        var resizeEvent = jQuery.Event('resize');
+
+        angular.element($window).trigger(resizeEvent);
```
Comment (myabc) [inline app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js]: @hschink can you try passing a String as argument? `angular.element($window).trigger('resize');`

Comment (hschink) [inline app/assets/javascripts/angular/work_packages/directives/options-dropdown-directive.js]: @myabc Please excuse that I didn't get you right away. I adapted the call accordingly.

**Comment by myabc** [comment]
:+1:

**Pull request merged**